src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -0,0 +1,2583 @@
+  "modifier.tconstruct.raging.flavor": "一个绝望的解决方案！",
+  "modifier.tconstruct.raging.description": "当持有者的生命值较低时,工具会造成更多伤害",
+  "modifier.tconstruct.raging.attack_damage": "狂暴伤害",
+  "modifier.tconstruct.enhanced": "光彩照人",
+  "modifier.tconstruct.enhanced.flavor": "很好看但用处不大！",
+  "modifier.tconstruct.enhanced.description": "玫瑰金搭配奖励升级非常棒！",
+  "modifier.tconstruct.airborne": "飞人",
+  "modifier.tconstruct.airborne.flavor": "飞上天空！",
+  "modifier.tconstruct.airborne.description": "在空中挖掘时工具不再受到惩罚",
+
+  "__comment": "Tier 3 Traits",
+  "modifier.tconstruct.overcast": "过分加厚",
+  "modifier.tconstruct.overcast.flavor": "覆盖面广！",
+  "modifier.tconstruct.overcast.description": "末地史莱姆可以加厚这个工具",
+  "modifier.tconstruct.lacerating": "撕裂",
+  "modifier.tconstruct.lacerating.flavor": "锋利的边缘！",
+  "modifier.tconstruct.lacerating.description": "使目标流血,造成额外伤害",
+  "modifier.tconstruct.maintained": "崭新",
+  "modifier.tconstruct.maintained.flavor": "保持更锐利,更长！",
+  "modifier.tconstruct.maintained.description": "工具在耐久度较高时挖掘得更快",
+  "modifier.tconstruct.maintained.mining_speed": "崭新挖矿",
+  "modifier.tconstruct.lightweight": "轻便",
+  "modifier.tconstruct.lightweight.flavor": "就像碎片之刃！",
+  "modifier.tconstruct.lightweight.description": "工具的体积非常轻,让你可以更快地攻击和挖掘",
+  "modifier.tconstruct.tasty": "美味",
+  "modifier.tconstruct.tasty.flavor": "Om Nom Nom(拟声)",
+  "modifier.tconstruct.tasty.description": "味道好极了,你忍不住要咬几口你的工具",
+
+  "death.attack.tconstruct.bleed": "%1$s 流血致死",
+  "death.attack.tconstruct.bleed.item": "%1$s 因 %2$s 受伤流血而死",
+
+  "__comment": "Tier 4 Traits",
+  "modifier.tconstruct.overlord": "霸主",
+  "modifier.tconstruct.overlord.flavor": "富豪！",
+  "modifier.tconstruct.overlord.description": "工具将一些耐久度作为新的超粘液对象",
+  "modifier.tconstruct.momentum": "动量",
+  "modifier.tconstruct.momentum.flavor": "不够快,得快点！",
+  "modifier.tconstruct.momentum.description": "挖掘可以提高你的速度,只要你不停止",
+  "modifier.tconstruct.momentum.mining_speed": "动量速度",
+  "modifier.tconstruct.insatiable": "贪婪",
+  "modifier.tconstruct.insatiable.flavor": "尝尝鲜血！",
+  "modifier.tconstruct.insatiable.description": "在战斗中,每次击中敌人都会造成越来越多的伤害",
+  "modifier.tconstruct.insatiable.attack_damage": "动量伤害",
+  "modifier.tconstruct.conducting": "热传导",
+  "modifier.tconstruct.conducting.flavor": "热力学！",
+  "modifier.tconstruct.conducting.description": "当你着火时,工具会造成成倍的伤害",
+  "modifier.tconstruct.conducting.attack_damage": "着火伤害最大值",
+
+  "__comment": "Tier 5 Traits",
+  "modifier.tconstruct.enderporting": "末影传送",
+  "modifier.tconstruct.enderporting.flavor": "利用末影史莱姆的力量",
+  "modifier.tconstruct.enderporting.description": "工具将持有者传送到开采的方块或被杀死的实体",
+
+
+  "__comment": "Tier 2 Mod Compat Traits",
+  "modifier.tconstruct.dense": "致密",
+  "modifier.tconstruct.dense.flavor": "几乎没有变形",
+  "modifier.tconstruct.dense.description": "工具使用寿命更长,但更难修复",
+  "modifier.tconstruct.sharpweight": "沉重",
+  "modifier.tconstruct.sharpweight.flavor": "相对于哑重量",
+  "modifier.tconstruct.sharpweight.description": "地雷更快,但非常难以移动",
```

```suggestion
  "modifier.tconstruct.sharpweight.description": "挖掘更快，但极难移动",
```
**此处有机翻痕迹。**（mines此处为mine的动词，而非名词地雷）
如果使用了机器翻译，请务必认真对照原文检查全文。